gslib/commands/defacl.py

```diff
@@ -224,19 +208,27 @@ def get_gcloud_storage_args(self):
         )
 
     elif sub_command == 'set':
-      if (os.path.isfile(self.args[0])):
+      acl_file_or_predefined_acl = self.args.pop(0)
+      if (os.path.isfile(acl_file_or_predefined_acl)):
         gcloud_storage_map = GcloudStorageMap(
           gcloud_command=[
             'storage', 'buckets', 'update',
-            '--default-object-acl-file=' + self.args.pop(0)
+            '--default-object-acl-file=' + acl_file_or_predefined_acl
           ],
           flag_map={},
         )
-      else:
+      else: # check if is valid predefined ACL string
```

Comment:
I think you can remove this comment, but, if you keep it, please capitalize and add a period